lsdpack.cpp

```diff
@@ -187,11 +191,13 @@ void record_gbs(int argc, char* argv[]) {
     }
 }
 
-void record_dump(int argc, char* argv[]) {
+void record_dump(int argc, char* argv[], unsigned flags) {
     writer = new DumpWriter();
 
     for (; optind < argc; ++optind) {
-        load_gb(argv[optind]);
+        load_gb(argv[optind], flags);
+        press(B);
+        press(0);
```

Comment:
From the look of it, these extra keypresses could be moved into load_gb.
Also, I think it would look nice with a comment, why those extra keypresses are needed.